.github/scripts/build-base-image.sh

```diff
@@ -29,16 +29,16 @@ case "${BASE_TAG}" in
         ;;
 esac
 
-echo "::set-output name=BASE::${BASE}"
-echo "::set-output name=BASE_TAG::${BASE_TAG}"
-echo "::set-output name=BASE_TYPE::${BASE_TYPE}"
+echo "BASE=$BASE" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
echo "BASE=${BASE}" >> ${GITHUB_OUTPUT}
```

Would you mind changing variable access to use this syntax so that it follows the style of the rest of these files?